docs/gno-tooling/cli/gnokey/full-security-tx.md

```diff
@@ -0,0 +1,128 @@
+---
+id: full-security-tx
+---
+
+# Making an airgapped transaction
+
+## Prerequisites
+
+- **`gnokey` installed.** Reference the
+  [Local Setup](../../../getting-started/local-setup/installation.md#2-installing-the-required-tools) guide for steps
+
+## Overview
+
+`gnokey` provides a way to create a transaction, sign it, and later
+broadcast it to a chain in the most secure fashion. This approach, while more 
+complicated, grants full control and provides airgap support.
+
+The indented purpose of this functionality is to provide maximum security when 
+signing and broadcasting a transaction. In practice, this procedure should take
+place on two separate machines, one with access to the internet (`Machine 1`), 
+and the other one without (`Machine 2`), with the separation of steps as follows:
+1. `Machine 1`: Fetch account information from the chain
+2. `Machine 2`: Create an unsigned transaction locally
+3. `Machine 2`: Sign the transaction
+4. `Machine 1`: Broadcast the transaction
+
+For the sake of simplicity, in this example, we will assume that the procedure 
+is happening on two machines, and we will again use the Userbook 
+realm on the Portal Loop testnet.
+
+## Fetching account information from the chain
+
+First, we need to fetch data for the account we are using to sign the transaction,
+using the [auth/accounts](./querying-a-network.md#authaccounts) query:
+
+```bash
+gnokey query auth/accounts/<your_address> -remote "https://rpc.gno.land:443"
+```
+
+We need to extract the account number and sequence from the output:
+
+```bash
+height: 0
+data: {
+  "BaseAccount": {
+    "address": "g1zzqd6phlfx0a809vhmykg5c6m44ap9756s7cjj",
+    "coins": "10000000ugnot",
+    "public_key": null,
+    "account_number": "468",
+    "sequence": "0"
+  }
+}
+```
+
+In this case, the account number is `468`, and the sequence (nonce) is `0`. We
+will need these values to sign the transaction later.
+
+## Creating an unsigned transaction locally
+
+To create the transaction you want, you can use the aforementioned `call` API,
```

Comment:
Where it is aforementioned? Is it on another page? Maybe a link or some context would help.

Reply:
Good catch. It used to be one big tutorial. Linked it: [8d6fd2a](https://github.com/gnolang/gno/pull/2226/commits/8d6fd2aa25b26c719c309cdfba83f192e3fa70ab)